docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -150,81 +150,81 @@ values requiring extrapolation being masked.
+using a linear interpolation scheme. To achieve this we pass the ``rotated_psl``
+cube to the regridder to supply the grid to regrid the ``global_air_temp`` cube onto:
 
     >>> rotated_air_temp = global_air_temp.regrid(rotated_psl, iris.analysis.Linear())
 
 .. plot:: userguide/regridding_plots/regridded_to_rotated.py
 
-Of course, we could have interpolated the pressure values onto the global grid, but
-this will involve some form of extrapolation. As with interpolation, it is in the
-definition of the scheme where the extrapolation mode can be controlled.
+We could regrid the pressure values onto the global grid, but this will involve
+some form of extrapolation. As with interpolation, we can control the extrapolation
+mode when defining the regridding scheme.
 
-When regridding the pressure cube, which is defined on a limited area rotated pole grid, on to
-the global grid as defined by the temperature cube, any linearly extrapolation
-values would quickly become dominant and highly inaccurate. We may therefore define the
-``extrapolation_mode`` in the constructor of :class:`iris.analysis.Linear` masking values which
-lie outside of the domain of the rotated pole grid:
+The ``rotated_psl`` cube is defined on a limited area rotated pole grid. If we regridded
+the ``rotated_psl`` cube onto the global grid as defined by the ``global_air_temp`` cube
+any linearly extrapolated values would quickly become dominant and highly inaccurate.
+We can control this behaviour by defining the ``extrapolation_mode`` in the constructor
+of the regridding scheme to mask values that lie outside of the domain of the rotated
+pole grid:
 
     >>> scheme = iris.analysis.Linear(extrapolation_mode='mask')
     >>> global_psl = rotated_psl.regrid(global_air_temp, scheme)
 
 .. plot:: userguide/regridding_plots/regridded_to_global.py
 
-Notice that, although we can still see the approximate shape of the rotated pole grid, the
-cells have now become rectangular in a plate-carrée/equirectangular projection, and that
-the resulting cube is really global, with a large proportion of the data being masked.
+Notice that although we can still see the approximate shape of the rotated pole grid, the
+cells have now become rectangular in a plate carrée (equirectangular) projection.
+The spatial grid of the resulting cube is really global, with a large proportion of the
+data being masked.
 
 Area weighted regridding
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-To conserve quantities when regridding, it is often the case that a point-based
-interpolation such as that provided by :class:`iris.analysis.Linear` is not
-appropriate. The :class:`iris.analysis.AreaWeighted` scheme is less general than
-:class:`iris.analysis.Linear`, but it is a conservative regridding scheme meaning
-that the area weighted total is approximately preserved across grids.
+It is often the case that a point-based regridding scheme (such as
+:class:`iris.analysis.Linear`) is not appropriate when you need to conserve
```

Comment:
... and now we do have an explicit mention of both `Linear` and `Nearest` :wink: